.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav.vtl

```diff
@@ -0,0 +1,812 @@
+#*
+ * OmniNav v2 format.
+ * Chapman.edu/_components/omni_nav_v2/omni_nav.vtl
+ *
+ * It would make sense to move the various component-specific macros (like those
+ * for Primary Nav) into their own file and import them like so:
+ *
+ * #import( "/_components/omni_nav_v2/macros/primary_nav.vm" )
+ *
+ * But this makes debugging the code in Cascade's format editor much harder. It is
+ * simpler to just keep it all in a well-structured single file.
```

Comment:
I would love to make this more modular but found it impractical for development.

I added a Velocity Best Practices wiki page:

- https://wimops.chapman.edu/wiki/Velocity